drivers/AnalogIn.h

```diff
@@ -126,3 +128,5 @@ class AnalogIn {
 #endif
 
 #endif
+
+/** @}*/
```

Comment:
My script did not automatically find the close-brace of the namespace, and doing so without having a proper parser would be difficult. (I just used a regexp)